apps/server/src/modules/user/legacy/interfaces/user-sort.query.ts

```diff
@@ -0,0 +1,5 @@
+import { SortOrderNumberType } from '@shared/domain/interface';
+
+export interface UserSortQuery extends SortOrderNumberType {
+	searchQuery?: number;
```

Comment:
This prop irritates me. Why does it exist?

Reply:
This is the only place, where it is used, right? https://github.com/hpi-schul-cloud/schulcloud-server/blob/e876b1e9b8215884481bdb56621e7c33fad07a1a/apps/server/src/modules/user/legacy/repo/helper/aggregation-helper.ts#L280

I know, you basically copied the code. I guess you chose type "number" because it was set to 1 in the old AdminUsers service, right?

Reply:
yeah, also it is sort value

Reply:
How do you mean, it is also sort value?

Reply:
I mead that is it number because it is sort value provided to mongo (sort order value)

Reply:
But where is this value provided to mongo?

Reply:
inside the aggregation query in sort part

Reply:
Can you link a line?

Reply:
search-query-helper.ts:15
https://github.com/hpi-schul-cloud/schulcloud-server/pull/4776/files#diff-a003cc4de725e5a3722d7659082e96712fe3d0908561086f44bd065c5aeb8da1R15
There is a test for that scenario

Reply:
Okay, but is there another place where it is read, except the line I linked above? There it makes no sense to have this as a numeric value. And also the name seems wrong for the usage. A boolean with a name like "isMetaTextScoreApplied" would seem appropriate to me. But I don't know if you should change it now or keep it in mind for the next ticket...

Reply:
You are right, I have changed the name of this property. Type will be changes after refactoring whole API.